src/Tes.RunnerCLI/Commands/HandlerEx.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.CommandLine.Binding;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+
+namespace Tes.RunnerCLI.Commands
+{
+    // TODO: This will need to be updated or removed when System.CommandLine is updated.
+
+    /// <summary>
+    /// Extends <see cref="Handler"/>
+    /// </summary>
+    /// <remarks>
+    /// This will need to be updated or removed when System.CommandLine is updated.
+    /// </remarks>
+    internal static class HandlerEx
```

Comment:
I don't think this is needed. We only have three commands, and likely won't have more in the future. The existing approach to set the handler is simpler in my opinion.

Reply:
This is needed when adding more options and/or arguments to any command such that the total number, including globals, sums past 8, unless you want to lose the pattern you are already using. This simply extends the same pattern already in the third party library to work for more than 8 symbols per command. When I added to the commands, it became necessary. Since it's already written and tested, we can either leave it in if we end up needing it, we can remove the file from Compile so that it's not included until we realize we do need it, or we can dump it and recreate it later when/if that becomes necessary.